.github/workflows/docfx.yml

```
@@ -0,0 +1,53 @@
+name: Deploy DocFX to GitHub Pages
+
+on:
+  push:
+    branches:
+      - feature/api-reference
+  release: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)